pages/index.js

```diff
@@ -1,7 +1,40 @@
+import Image from 'next/image';
+
 export default function Home() {
+  console.log('hello');
   return (
-    <h1 className="text-3xl font-bold underline">
-      Hello world!
-    </h1>
+    <div>
+      <nav class="bg-white border-gray-200 px-2 sm:px-4 py-2.5 rounded dark:bg-gray-900">
+        <div class="container flex flex-wrap items-center justify-between mx-auto">
+          <a href="#" class="flex items-center">
+              <Image src="/./hartslogo.png" class="h-6 mr-3 sm:h-9" alt="Hart's Logo" width="100" height="100" />
+              <span class="self-center text-xl font-semibold whitespace-nowrap dark:text-white">Hart&apos;s</span>
```

Comment:
This is not matching the design, the design shows the logo, not a text or the logo with a text.

Your design:
![Screenshot 2022-12-16 at 21 12 20](https://user-images.githubusercontent.com/45739963/208218739-e7c47777-4a97-43b0-9d3b-13a9cc2ed41d.png)

The required design:
![Screenshot 2022-12-16 at 21 13 27](https://user-images.githubusercontent.com/45739963/208218794-40f9da2f-b6f1-4065-a615-1c39f6061ea2.png)

Also use color picker to adjust color